cli/src/pcluster/resources/compute_node/user_data.sh

```diff
@@ -151,6 +151,18 @@ write_files:
         done;
         export HOME="${!HOME_BAK}"
       }
+
+      function publish_startup_time
+      {
+        StartTime=$1
```

Comment:
let's avoid camelcase variables in bash.

Since they are internal variables let's use lowercase.

Reply:
Done